toxcore/query.h

```diff
@@ -0,0 +1,57 @@
+/** query.h
```

Comment:
Redundancy of information is to be avoided, because it increases maintenance load. Renaming the file may render this line out of date.

Besides that, I don't need to give you a reason *not* to include useless information. It is your obligation to give a reason *for* doing it.